### PR TITLE
Meeting contributors

### PR DESCRIPTION
**Pull request by warrd**
### Issue

https://github.com/wellcometrust/platform/issues/3632

### Description

Add 111 and 711 to contributors as `Meeting`

### CODE REVIEW

File: pipeline/transformer/transformer_sierra/src/main/scala/uk/ac/wellcome/platform/transformer/sierra/transformers/SierraContributors.scala [hunk 1]
```diff
@@ -37,82 +46,92 @@ object SierraContributors
...
+        )
       }
+    }
   }
 
   private def getContributionRoles(
-    subfields: List[MarcSubfield]): List[ContributionRole] = {
-    // Extract the roles from subfield $e.  This is a repeatable field.
+    subfields: List[MarcSubfield],
+    subfieldTag: String = "e"): List[ContributionRole] =
```
Comment (jamesgorrie) [inline pipeline/transformer/transformer_sierra/src/main/scala/uk/ac/wellcome/platform/transformer/sierra/transformers/SierraContributors.scala]: Is this a logical default, or just to help with backwards compatibility?

File: pipeline/transformer/transformer_sierra/src/test/scala/uk/ac/wellcome/platform/transformer/sierra/transformers/SierraContributorsTest.scala [hunk 2]
```diff
@@ -573,6 +582,85 @@ class SierraContributorsTest
       expectedContributors = List())
   }
 
+  describe("Meeting") {
+    it("gets the name from MARC tag 111 subfield $$a") {
+      val varField = createVarFieldWith(
+        marcTag = "111",
+        subfields = List(MarcSubfield(tag = "a", content = "Big meeting"))
```
Comment (jamesgorrie) [inline pipeline/transformer/transformer_sierra/src/test/scala/uk/ac/wellcome/platform/transformer/sierra/transformers/SierraContributorsTest.scala]: Sounds very important.

File: pipeline/transformer/transformer_sierra/src/main/scala/uk/ac/wellcome/platform/transformer/sierra/transformers/SierraContributors.scala (hunk 1 above)
Comment (warrd) [reply pipeline/transformer/transformer_sierra/src/main/scala/uk/ac/wellcome/platform/transformer/sierra/transformers/SierraContributors.scala]: This was uses by the previous two (`organisation` and `person`). I could remove the default and make it explicit if you think makes more sense?

File: pipeline/transformer/transformer_sierra/src/main/scala/uk/ac/wellcome/platform/transformer/sierra/transformers/SierraContributors.scala [hunk 3]
```diff
@@ -37,82 +46,92 @@ object SierraContributors
...
-        val maybeAgent = getOrganisation(subfields)
+  private def getMeetingContributors(
+    bibData: SierraBibData,
+    marcTag: String): List[Contributor[MaybeDisplayable[Meeting]]] = {
+    getMatchingSubfieldLists(
+      bibData,
+      marcTag = marcTag,
+      marcSubfieldTags = List("a", "c", "d", "j", "t", "0")
+    ).flatMap { subfields: List[MarcSubfield] =>
+      val roles = getContributionRoles(subfields, "j")
```
Comment (jamesgorrie) [inline pipeline/transformer/transformer_sierra/src/main/scala/uk/ac/wellcome/platform/transformer/sierra/transformers/SierraContributors.scala]: [This in the spec](https://www.loc.gov/marc/bibliographic/bd111.html) says "Relator term" <= is that a role?

File: pipeline/transformer/transformer_sierra/src/main/scala/uk/ac/wellcome/platform/transformer/sierra/transformers/SierraContributors.scala (hunk 1 above)
Comment (jamesgorrie) [reply pipeline/transformer/transformer_sierra/src/main/scala/uk/ac/wellcome/platform/transformer/sierra/transformers/SierraContributors.scala]: Having these funny floaty letters always feels odd, I would be explicit as there's nothing in the spec or elsewhere saying there is any consistency with Marc...

File: pipeline/transformer/transformer_sierra/src/main/scala/uk/ac/wellcome/platform/transformer/sierra/transformers/SierraContributors.scala (hunk 3 above)
Comment (jamesgorrie) [reply pipeline/transformer/transformer_sierra/src/main/scala/uk/ac/wellcome/platform/transformer/sierra/transformers/SierraContributors.scala]: Sorry, me being silly, missed it on Person...

**Pull request merged**